assessment/collectors/project_analyzer/java_file_dependency_analyzer.go

```diff
@@ -0,0 +1,336 @@
+package assessment
```

Comment:
Can you compare the dependency graph generated from this code with Java dependency graph generated from the python code?

Reply:
I compared it with the Python Code, it's similar to this PR with few differences:
Python code has the same exceptions with 2 additional exception that I could find:
1. If 2 classes with same name exists with the Project, then only 1 will be considered.
2. Extends and Implements are not considered.